l10n: ko.po: Fix typo

elongbug opened the pull request:
- Fix three typos of Korean translation.

elongbug committed:
l10n: ko.po: Update Korean translation

Fix eight fuzzy Korean translations.
Add Gwan-gyeong Mun <elongbug@gmail.com> as a contributor.

Signed-off-by: Gwan-gyeong Mun <elongbug@gmail.com>

elongbug committed:
l10n: ko.po: Fix typo

Fix three typos of Korean translation.

Signed-off-by: Gwan-gyeong Mun <elongbug@gmail.com>

changwoo commented on po/ko.po:
```diff
@@ -9559,16 +9551,12 @@ msgid "Use mail map file"
-#| msgid "only use refs matching <pattern>"
 msgid "only decorate refs that match <pattern>"
-msgstr "<패턴>과 일치하는 레퍼런스만 사용합니다"
+msgstr "<패턴>과 일치하는 레퍼런스만 꾸미게 합니다"
```
불필요한 사동: "꾸미게 합니다" -> "꾸밉니다"

elongbug committed:
l10n: ko.po: Update Korean translation

Add a term of Korean translation: fsmonitor.
Fix eleven fuzzy Korean translations.

Signed-off-by: Gwan-gyeong Mun <elongbug@gmail.com>

elongbug committed:
l10n: ko.po: Update Korean translation

Fix three typos of Korean translation.
Add sixteen Korean translation.

Signed-off-by: Gwan-gyeong Mun <elongbug@gmail.com>
